cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -287,12 +292,13 @@ private static String findNetwork(String project, Compute compute) throws IOExce
       networks = networkList.getItems();
     } catch (Exception e) {
       handleRetryableExceptions(e);
-      throw e;
+      throw new ErrorCodeWrappedException(e, ErrorGroup.DATAPROC, ErrorCodeProvider.ERR_NETWORK);
     }
 
     if (networks == null || networks.isEmpty()) {
-      throw new IllegalArgumentException(String.format("Unable to find any networks in project '%s'. "
-                                                         + "Please create a network in the project.", project));
+      throw new InvalidConfigurationException(String.format("Unable to find any networks in project '%s'. "
+                                                         + "Please create a network in the project.", project),
+              ErrorGroup.DATAPROC, ErrorCodeProvider.ERR_NETWORK);
```

Comment:
style: indentation should be consistent with the rest of the project (aligned with the argument from the previous line)

Reply:
Done.